### ISSUE TITLE
Improve `VaAlert`

### ISSUE THREAD

**Issue by Derranion** [epicmaxco/vuestic-ui#1694]
**Subject**:
Currently there is no straightforward way to control  `text` color and `background-color` of `VaAlert` for the users.

**Example**:
```
<va-alert color="info" title="Title">
  No idea which color text is.
</va-alert>
```
![AlerColorInfo](https://user-images.githubusercontent.com/7745899/164498284-1f84c7f3-5a0f-4546-a91b-28a7b48eb5cb.png)

Now lets add an `outline`:

![AlertOutline](https://user-images.githubusercontent.com/7745899/164502421-14721940-c516-4bad-a277-39fabfe65a49.png)

Theme has been changed but users may want to have different colors in some of their `Alerts`.

Adding `border` will change text color again.
![outlineAndBorder](https://user-images.githubusercontent.com/7745899/164509135-b974cccd-35d8-4dc6-85cf-91258204422d.png)

**Summary**:
In other words there are 3 different themes that are linked with these props.
Users may want to utilize the props but keep the same color as before or even set the new custom one.

I believe that `local config` is not the best solution for such a common thing and it will not resolve this completely.
That's why we could add `prop` for the `text` color and `background-color`.

**Ideas to discuss:**

- [ ] `Text` color property
- [ ] `Background` color property

**Optional**:
- [x] Add an option to change outline color
- [ ] Add an option to set border from all sides (it is different from `outline`)
- [ ] Add warning when `border-color` prop is used without the `border` prop

**Comment by asvae** [comment]
See also https://github.com/epicmaxco/vuestic-ui/issues/1729
